core/document/src/document.rs

```diff
@@ -197,6 +197,24 @@ impl Document {
 
 				Some(vec![DocumentResponse::DocumentChanged])
 			}
+			Operation::AddEllipse {
+				path,
+				insert_index,
+				cx,
+				cy,
+				rx,
+				ry,
+				rot,
+				style,
+			} => {
+				self.add_layer(
+					&path,
+					Layer::new(LayerDataTypes::Ellipse(layers::Ellipse::new(kurbo::Point::new(cx, cy), kurbo::Vec2::new(rx, ry), rot, style))),
```

Comment:
since `layers::Ellipse::new` uses `impl Into<kurbo::{Point, Vec2}>`, this code could be made simpler by omitting those `new` calls in favor of `(f64, f64)`: 
```
layers::Ellipse::new((cx, cy), (rx, ry), …)
```
This change should then also be made in the other match arms.
If not, we should remove the `impl Into` from those `new` function's arguments.
What do you think, @Keavon?

Reply:
The tuple form makes it shorter and easier to read.